app/models/vm.rb

```diff
@@ -121,6 +121,12 @@ def supported_consoles
     }
   end
 
+  def v2v_migrated?
```

Comment:
The purpose of the method is to identify whether the VM can be added to a plan. A valid VM is the one has  `ServiceResource#status` as `Failed` or never exists in `ServiceResource`.

Also reconsider the method name.